web_src/js/features/emoji.js

```diff
@@ -0,0 +1,23 @@
+import emojis from '../../../public/emoji/emoji.json';
+
+const {StaticUrlPrefix} = window.config;
+
+export const giteaImage = `<img alt=":gitea:" title=":gitea:" class="emoji" src="${StaticUrlPrefix}/img/emoji.png" align="absmiddle">`;
+
+const tempMap = {gitea: 'gitea'};
+for (const {emoji, aliases} of emojis) {
```

Comment:
I still want to have access to the description field as before I think it makes a better text result in the menu but more importantly it makes a better aria-label and I want to use it for that

Reply:
I specifically changed it to use the keyword because that matches GitHub's behaviour. I think the descriptions are not really useful and would mostly just be confusing. Everyone seems to refer to emojis based on the description.

I think it may be nice to have `title="keyword"` tooltip, that's something I've wanted on GitHub too.

Reply:
Sorry, I like the description better for at least the aria label here and do want to have access to it here as it was before.